charts/engine/templates/engine-job-config.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: engine-job-config
+data:
+{{ if .Values.gpu }}
+  defaultGpuType: engine-job-template-k80-cm
+{{ else }}
+  defaultGpuType: engine-job-template-pysvcs-cm
+{{ end }}
```

1. you should reference the `ids`, not the templates
2. I think it can be written with a ternary expression
```suggestion
  defaultGpuType: {{ .Values.gpu | ternary "gpu" "cpu" }}
```